docs/developer/extending/apps/developing-apps/app-sdk/protected-views.md

```diff
@@ -0,0 +1,61 @@
+# App-protected views
```

Comment:
**Suggestion[^1][^2]**:
Maybe a "Protecting the App dashboard interface" would be a better name?

[^1]: ✅ Not blocking the review. 
[^2]: It might have an impact on the submitted code. I've probably had a similar issue with the chosen approach. If you're sure that's overkill or does not apply to this PR, let me know and resolve the request.

Reply:
This one displays in the sidebar, I would rather keep it a bit shorter.